event-store/build.gradle.kts

```diff
@@ -16,7 +16,7 @@ tasks.test {
 
 dependencies {
   implementation("com.typesafe:config:1.3.2")
-  implementation("org.apache.kafka:kafka-clients:5.5.0-ccs") {
+  implementation("org.apache.kafka:kafka-clients:7.2.1-ccs") {
```

Comment:
please use the kafka-bom here. Waiting for a review on https://github.com/hypertrace/kafka-streams-framework/pull/63

Also I'm guessing this repo needs to update to use the owasp dependency check, example can be found here: https://github.com/hypertrace/document-store/pull/158

Reply:
So far so good, next part would be to update the CI run so it runs the new scan instead of the old snyk scan